deploy/role.yaml

```diff
@@ -32,6 +32,7 @@ rules:
   verbs:
   - get
   - create
+  - list
```

Comment:
I think we need this one also under the config folder so it will be generated for the bundle

Reply:
the contents of the `config/` folder is currently out of sync with `openshit/sriov-network-operator` (e.g. see `config/rbac/role.yaml` [here](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/config/rbac/role.yaml) and [here](https://github.com/openshift/sriov-network-operator/blob/master/config/rbac/role.yaml)).

I would align that in a separated PR. Does it sound good?